src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Square implements Figure {
+    private int side;
+    private String color;
+    public Square() {
+        this.side = side;
+        this.color = color;
+    }
```

Comment:
The class `Square` should have a method `draw()` to adhere to the `Figure` interface contract if the `draw()` method is a part of it. Make sure to implement all abstract methods from the interface.